x-pack/plugins/security_solution/public/management/pages/policy/view/components/antivirus_registration_form/index.tsx

```diff
@@ -52,9 +53,26 @@ export const AntivirusRegistrationForm = memo(() => {
       }),
     [dispatch]
   );
-
   return (
-    <ConfigForm type={TRANSLATIONS.title} supportedOss={[OperatingSystem.WINDOWS]}>
+    <ConfigForm
+      type={TRANSLATIONS.title}
+      supportedOss={[OperatingSystem.WINDOWS]}
+      osRestriction={
+        <>
```

Comment:
I found that when this is displayed in Fleet, the tooltip wraps to its own line:
![image](https://user-images.githubusercontent.com/56442535/109720802-8dc93d80-7b78-11eb-8560-2d99ec0c9f52.png)


Suggestion: breach this out to a component (instead of inline here) and wrap the two element below in Flex components that don't grow or wrap.